lib/manual_relocator.rb

```diff
@@ -224,6 +220,30 @@ def send_draft(manual, manual_edition, document_editions)
     end
   end
 
+  def send_gone(document_id, slug)
+    # We should be able to use
+    #   publishing_api.unpublish(document_id, type: 'gone')
+    # here, but that doesn't leave the base_path in a state where
+    # publishing_api will let us re-use it.  Sending a draft gone object
+    # and then publishing it does though.  Might want to check if we can
+    # go back to the unpublish version at some point though.
```

Comment:
We've got a card for this on our planning board: https://trello.com/c/4RRvSMeQ/316-update-substitution-helper